src/general/Species.cs

```diff
@@ -79,6 +79,13 @@ protected Species(uint id, string genus, string epithet)
     [JsonIgnore]
     public string FormattedName => Genus + " " + Epithet;
 
+    /// <summary>
+    ///   Returns <see cref="FormattedName"/> but added with bbcode tags. Player's species will be emphasized with
```

Comment:
```suggestion
    ///   Returns <see cref="FormattedName"/> but includes bbcode tags for styling. Player's species will be emphasized with
```
Sounds a bit more natural to me